sklearn/multiclass.py

```diff
@@ -74,6 +74,14 @@ def _fit_binary(estimator, X, y, classes=None):
     return estimator
 
 
+def _partial_fit_binary(estimator, X, y, classes=None):
+    """Partially fit a single binary estimator."""
+    if (not hasattr(estimator, "partial_fit")):
+            raise ValueError("Estimator must have partial_fit method")
```

Comment:
It would be great to display something about the estimator (for instance the repr of this estimator) in the error message. It really helps users figure out what is wrong.